src/views/run-new.js

```diff
@@ -0,0 +1,16 @@
+export default Backbone.View.extend({
+  tagName: 'new'
```

Comment:
Missing a `,` here after you declare the `tagName` property. This is giving me an error `Unexpected token (5:0)`